VideoLocker/src/main/java/org/edx/mobile/services/EdxCookieManager.java

```diff
@@ -53,8 +53,6 @@ public void onReceiveValue(Object value) {
         }
         PrefManager pref = new PrefManager(MainApplication.instance(), PrefManager.Pref.LOGIN);
         pref.put(PrefManager.Key.AUTH_ASSESSMENT_SESSION_ID, "");
-        //we can not get the expiration date from cookie, so just set it to expire for one day
-        pref.put(PrefManager.Key.AUTH_ASSESSMENT_SESSION_EXPIRATION, -1);
```

Comment:
I thought about that, but it's simple enough logic that it seemed like we could add it back in when we got to that and I like not having code sitting around doing nothing if we can avoid it. Also, it doesn't necessarily need to be persistent across runs of the app and has slightly different semantics, so it seemed better to start fresh.